views/journal.handlebars

```diff
@@ -0,0 +1,24 @@
+<section class="block">
+    <h2 class="p-3 is-family-secondary is-size-3 has-text-centered m-4">{{title}}</h2>
+    <section class="columns is-centered justify-content-center">
+        <div class="column justify-content-center has-background-grey-lighter p-2">
+            {{#unless entries}}
+            <p class="has-text-centered is-size-5 is-family-secondary">This journal has no entries! Would you like to create one?</p>
+            {{/unless}}
+
+            <button id="create-entry-button" class="button is-primary is-outlined my-3">Create New Entry</button>
+
+            {{#if entries}}
+            {{>entries}}
+            {{/if}}
+        </div>
+
```

Comment:
is this section to render one specific entry on the side to view? if so, i think we'll need that in it's own div so we can do a flexbox or something to align them side by side